seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -1596,7 +1596,7 @@
         "Here's an example of a radio button:",
         "<code>&#60;label&#62;&#60;input type=\"radio\" name=\"indoor-outdoor\"&#62; Indoor&#60;/label&#62;</code>",
         "<hr>",
-        "Add a pair of radio buttons to your form. One should have the option of <code>indoor</code> and the other should have the option of <code>outdoor</code>."
+        "Add a pair of radio buttons to your form. One should have the option of <code>indoor</code> and the other should have the option of <code>outdoor</code>. All two should share the <code>name</code> attribute of <code>indoor-outdoor</code>"
```

Comment:
@adetime Fair enough. I think `Both should` is fine, `buttons` can be omitted. I'm not a native speaker either, so I get where you're coming from 🙂